x/exchange/spec/01_concepts.md

```diff
@@ -208,18 +234,36 @@ Orders with external ids can be looked up using the [GetOrderByExternalID](05_qu
 External ids are limited to 100 characters.
 
 
+
+## Commitments
+
+A Commitment allows an account to give control of some of its funds to a market.
+
+When funds are committed to a market, they remain in the source account and a hold is placed on them.
```

Comment:
We are redefining the `Hold` module here. It's probably better to redirect.

Reply:
Not redefining anything here, providing context and specifics that the hold module doesn't know anything about. But I did turn "hold" into a link to the hold spec docs section describing holds.

Reply:
If the link was not included, then hold would need to be redefined. Changes are good.